resources/services/sagemaker/training_jobs.go

```diff
@@ -848,10 +859,12 @@ func resolveSagemakerTrainingJobTags(ctx context.Context, meta schema.ClientMeta
 }
 func resolveSagemakerTrainingJobSecondaryStatusTransitions(_ context.Context, _ schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
 	r, ok := resource.Item.(*sagemaker.DescribeTrainingJobOutput)
-
 	if !ok {
 		return fmt.Errorf("expected DescribeEndpointConfigOutput but got %T", resource.Item)
+	} else if r == nil {
+		return nil
 	}
+
```

Comment:
```suggestion
	if r == nil {
		return nil
	}

```

Reply:
if..elseif chains with every "upper" block always returning isn't that confusing, is it? otherwise linter would've caught that.

Reply:
Its a a nitpick, I find it harder to look at, and lets less letters, but ill accept also else-if, either way it should be consistent across the whole repo.